src/app-layout/widget.tsx

```diff
@@ -0,0 +1,20 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+import { getGlobalFlag } from '../internal/utils/global-flags';
+import { AppLayoutProps } from './interfaces';
+import { useVisualRefresh } from '../internal/hooks/use-visual-mode';
+import { AppLayoutImplementation } from './implementation';
+
+type AppLayoutType = React.ForwardRefExoticComponent<AppLayoutProps & React.RefAttributes<AppLayoutProps.Ref>>;
+
+export function createWidgetizedLayout(AppLayoutLoader?: AppLayoutType): AppLayoutType {
```

Comment:
non-blocking: we might want to name this function "createWidgetizedAppLayout" instead of "createWidgetizedLayout", to align it with the naming of the equivalent function for the split panel

Reply:
Good point, updated